### PR TITLE
Completed html-web-page exercise. - Barbara Liao

### PR DESCRIPTION
**Pull request by barbara-liao**
html-web-page screenshot

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    monthly basis. For the jelly lover in your life (or yourself!), this is the gift that keeps on giving all year round. We offer
+    three tiers of monthly jelly joy, and each one offers their own perks. Here is a breakdown of each, as well as your monthly price:
+  </p>
+  <table>
+    <tr>
+      <th></th>
+      <th>Basic Plan</th>
+      <th>Deluxe Plan</th>
+      <th>Jelly Giant Plan</th>
+    </tr>
```
Comment (codingcodymiller) [inline html-web-page/index.html:34]: This should be in a `thead`

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <td></td>
+      <td></td>
+      <td>&check;</td>
+    </tr>
+    <tr>
+      <td>Monthly Pricing</td>
+      <td>$30</td>
+      <td>$50</td>
+      <td>$60</td>
+    </tr>
```
Comment (codingcodymiller) [inline html-web-page/index.html:64]: This should be in a `tbody`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! A couple minor notes for you:

File: html-web-page/index.html (hunk 2 above)
Comment (barbara-liao) [reply html-web-page/index.html]: Whoops! Thanks for letting me know!

**Review by codingcodymiller — APPROVED** [review]
Great work!! 🎉

**Pull request merged**